DataFormats/Provenance/src/classes_def.xml

```diff
@@ -73,7 +73,8 @@
  <class name="edm::BranchChildren" ClassVersion="10">
   <version ClassVersion="10" checksum="137742168"/>
  </class>
- <class name="edm::ProductProvenance" ClassVersion="11">
+ <class name="edm::ProductProvenance" ClassVersion="12">
+  <version ClassVersion="12" checksum="737844716"/>
```

Comment:
Do we store `edm::ProductProvenance` in the present file format, or only in some old file format?

Reply:
From what I can tell, we do not store it now. I had changed the class without updating the iorule and only our backwards compatible file format test failed.